### ISSUE TITLE
Gradient vanishing of G in the DCGAN example

### ISSUE THREAD

**Issue by zhan4817** [pytorch/examples#822]
Hello,

I have trained the DCGAN with the default hyper-parameter settings on the downloaded "img_align_celeba" dataset (recommended in the tutorial). However, the results reveal strong gradient vanishing of G. While Loss_D keeps decreasing towards 0, Loss_G grows high (towards 100). 

It seems that D is trained so well, preventing a good training on G. I didn't do any modifications on the code. Do you know what happened?

Thanks!
